### ISSUE TITLE
Improve settings.ini

### ISSUE THREAD

**Issue by agoetz** [paesanilab/potential_fitting#68]
QM model section should be consistent across different steps. Different steps (e.g. nmcgen and database_filler) should be able to run independently and either with the same or different ini files.

**Comment by EthanBullVulpe** [comment]
when you say QM model section should be consistent, do you mean that we should have one method/basis for all steps instead of a separate method/basis for each?

**Comment by agoetz** [comment]
Yes, I think it may be better. Having to repeat the QM model section seems error prone. It seems to have confused the REHS students. I am not sure but it may be better to work with different settings files for different steps of the workflow, or have the option to do so. What do you think?
